dask_jobqueue/core.py

```diff
@@ -166,6 +168,7 @@ def __init__(
         death_timeout=None,
         local_directory=None,
         extra=None,
+        worker_command="distributed.cli.dask_worker",
```

Comment:
To be consistent with other kwargs, the default here should be None, and the true default defined in the configuration file. See how other kwargs are handled.

Reply:
Done!